src/l500/ac-trigger.cpp

```diff
@@ -681,12 +682,27 @@ namespace ivcam2 {
 
                     auto df = _sf.get_depth_frame();
                     auto irf = _sf.get_infrared_frame();
-                    depth_to_rgb_calibration algo( df, irf, _cf, _pcf, cal_info, cal_regs );
+
+                    depth_to_rgb_calibration algo(df, irf, _cf, _pcf, cal_info, cal_regs, 
+                    [&]() 
+                    {
+                        if (!is_processing())
+                        {
+                            AC_LOG( DEBUG , "The device has been disconnected during Camera Accuracy Health process");
```

Comment:
That's the only reason? What if the user simply turned off the depth stream?

Reply:
You are right, updated to "Stopping CAH algorithm"